Assets/Scripts/Control&Input/InputManager.cs

```diff
@@ -17,6 +17,8 @@ public class InputManager : MonoBehaviour
     public InputEvent onCancel;
     public InputEvent onMovePerformed;
     public InputEvent onMoveCanceled;
+    public InputEvent onLeftShoulderButtonPressed;
```

Comment:
I agree with the generalization to an action rather than specific button, I've renamed the variables to `onLeftTabPerformed` and `OnRightTabPerformed`, and similarly reflected the change in the inputMap.